tests/fixtures/migration/old_cassette.yaml

```diff
@@ -1,3 +1,6 @@
+# This cassette is problematic because it was recorded
```

Comment:
This cassette fails to deserialize at all in 3.7...it may be a good test for "bad yaml" if someone was moving from VCR<2.1 on python 2.7 to python 3 and vcr>=2.1 it doesn't produce a yaml loads error, just a format error.

If we want to keep this as valid yaml that will load we need to make the following patch:

```
diff --git a/tests/fixtures/migration/old_cassette.yaml b/tests/fixtures/migration/old_cassette.yaml
index 68d6b49..bd9c286 100644
--- a/tests/fixtures/migration/old_cassette.yaml
+++ b/tests/fixtures/migration/old_cassette.yaml
@@ -1,9 +1,6 @@
-# This cassette is problematic because it was recorded
-# with a Python 2.x environment and would NEVER deserialize
-# properly in a 3.x environment.
 - request: !!python/object:vcr.request.Request
     body: null
-    headers: !!python/object/apply:__builtin__.frozenset
+    headers: !!python/object/apply:builtins.frozenset
     - - !!python/tuple [accept-encoding, 'gzip, deflate, compress']
       - !!python/tuple [user-agent, python-requests/2.2.1 CPython/2.6.1 Darwin/10.8.0]
       - !!python/tuple [accept, '*/*']
@@ -13,9 +10,9 @@
     port: 80
     protocol: http
   response:
-    body: {string: !!python/unicode "{\n  \"origin\": \"217.122.164.194\"\n}"}
-    headers: [!!python/unicode "access-control-allow-origin: *\r\n", !!python/unicode "content-type:
-        application/json\r\n", !!python/unicode "date: Mon, 21 Apr 2014 23:06:09 GMT\r\n",
-      !!python/unicode "server: gunicorn/0.17.4\r\n", !!python/unicode "content-length:
-        32\r\n", !!python/unicode "connection: keep-alive\r\n"]
+    body: {string: !!python/str "{\n  \"origin\": \"217.122.164.194\"\n}"}
+    headers: [!!python/str "access-control-allow-origin: *\r\n", !!python/str "content-type:
+        application/json\r\n", !!python/str "date: Mon, 21 Apr 2014 23:06:09 GMT\r\n",
+      !!python/str "server: gunicorn/0.17.4\r\n", !!python/str "content-length:
+        32\r\n", !!python/str "connection: keep-alive\r\n"]
     status: {code: 200, message: OK}
```